index.src.html

```diff
@@ -691,16 +691,31 @@ <h3 id="user-mediation">
     Explicit user mediation
   </h3>
 
-  If a feature has privacy or security impacts that are endemic to the feature
-  itself, then one valid strategy for exposing it to the web is to require user
-  mediation before granting an origin access. For instance, [[GEOLOCATION-API]]
-  reveals a user's location, and wouldn't be particularly useful if it didn't;
+  If the security or privacy risk of a feature cannot otherwise be mitigated in
+  a specification, optionally allowing an implementer to prompt a user may
+  provide the best security/privacy possible.  If the specification does not
+  allow for the implementer to prompt, it may result in divergence
+  implementations by different user agents as some user agents choose to
+  implement more privacy-preserving version.
+
+  It is possible that the risk of a feature cannot be mitigated because the
+  risk is endemic to the feature itself.  For instance, [[ GEOLOCATION-API ]] 
+  reveals a user’s location, and wouldn’t be particularly useful if it didn’t;
```

Comment:
Today I would argue that it would be useful if providing course location details + information on precision. Should we change this part of the text not to endorse the default behavior of geolocation?

Reply:
Revised with https://github.com/w3cping/security-questionnaire/commit/64200400d6b4bd60a3294ab4e906ec5155c668eb to make clear this is about how the geolocation api works rather than opining on it.